### PR TITLE
Save function for push job object in etcd

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

File: tendrl/gluster_integration/central_store/__init__.py [hunk 1]
```diff
@@ -44,3 +44,6 @@ def save_nodecontext(self, node_context):
 
     def save_utilization(self, utilization):
         tendrl_ns.etcd_orm.save(utilization)
+
+    def save_job(self, job):
```
Comment (shtripat) [inline tendrl/gluster_integration/central_store/__init__.py]: Isnt this done as part another PR?

Comment (GowthamShanmugam) [inline tendrl/gluster_integration/central_store/__init__.py]: @shtripat yes job class in common right,  but save method should be in central_store to store that  job object, so i have created job class in common and save method in all component.

Comment (GowthamShanmugam) [inline tendrl/gluster_integration/central_store/__init__.py]: @shtripat if the component want to create any job, job is created using job class in common and stored via save method of its own (https://github.com/Tendrl/commons/pull/175)
